internal/serve/httphandler/assets_handler.go

```diff
@@ -69,12 +68,14 @@ func (c AssetsHandler) CreateAsset(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	assetCode := strings.TrimSpace(strings.ToUpper(assetRequest.Code))
+	assetIssuer := strings.TrimSpace(assetRequest.Issuer)
+
 	v := validators.NewValidator()
-	v.Check(assetRequest.Code != "", "code", "code is required")
-	if strings.ToUpper(assetRequest.Code) != stellarNativeAssetCode {
-		v.Check(assetRequest.Issuer != "", "issuer", "issuer is required")
-		assetRequest.Issuer = strings.TrimSpace(assetRequest.Issuer)
-		v.Check(strkey.IsValidEd25519PublicKey(assetRequest.Issuer), "issuer", "issuer is invalid")
+	v.Check(assetCode != "", "code", "code is required")
+	if assetCode != stellarNativeAssetCode {
+		v.Check(strkey.IsValidEd25519PublicKey(assetIssuer), "issuer", "issuer is invalid")
+		v.Check(assetIssuer != "", "issuer", "issuer is required")
```

Comment:
nit: just wondering if this check is required at all, if you're already checking whether it's a valid curve key above?

Reply:
Good point! It's not needed per se but I decided to keep it there because it's informative to know that the issuer is missing

On second thought, I agree with you and will remove it.

Reply:
Removed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/255/commits/77de66f04e4e1ca4d387fb2a0ca16ac212d3fffa.